sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/AssignRoleToUserTest.java

```diff
@@ -105,6 +105,7 @@ private void createUser() {
 
   private void createOrg() {
     if (StringUtils.isBlank((String) testContext.getVariables().get("organisationId"))) {
+      variable("rootChannel", OrgUtil.getRootChannel());
```

Comment:
In latest code getRootChannel is renamed to getRootOrgChannel(). Also, rename rootChannel to rootOrgChannel.